cves/CVE-2010-0425.yml

```diff
@@ -184,8 +201,10 @@ lessons:
     applies:
     note:
   yagni:
-    applies:
-    note:
+    applies: true
+    note: I feel like this lesson kind of applies here since the piece of code
```

Comment:
"feel like" and "kind of" should not both be here, I personally would put "Yagni applies here" as I feel like this is the perfect example of Yagni.

Reply:
Fixed wording in latest commit.